docs/pom.xml

```diff
@@ -36,6 +36,10 @@
 			<id>docs</id>
 			<build>
 				<plugins>
+					<plugin>
```

Comment:
Since we add the `maven-dependency-plugin` section in `pluginManagement`, we need to also set it up in the `docs` module, so that we do fetch and unpack the proper resources